docs/specs/clients/sign/client-api.md

```diff
@@ -74,6 +74,50 @@ abstract class Client {
     reason: Reason;
   }): Promise<void>;
 
+  // ---------- Wallet Authentication Methods ----------------------------------------------- //
+
+  // for proposer to request wallet authentication
+  public abstract sessionAuthenticate(params: {
+    chains: string[];
+    domain: string;
+    nonce: string;
+    aud: string;
+    type?: CacaoHeader["t"];
+    nbf?: string;
+    exp?: string;
+    statement?: string;
+    requestId?: string;
+    resources?: string[];
+    expiry?: number;
```

Comment:
what's the difference between `expiry` and `exp`?

Reply:
`expiry` to be removed, and `exp` to be used for both cacao expiry and request ttl

Reply:
> expiry to be used for request ttl

I think we should call it ttl and it's for calculating request expiry, not related to relay ttl, relay ttl can be always 5 minutes

Reply:
Yeah this is really confusing to have the same concept duplicated... TTL would be easier